embedded-workshop-book/src/dongle.md

```diff
@@ -61,6 +61,34 @@ deviceid=588c06af0877c8f2 channel=20 TxPower=+8dBm app=loopback.hex
 
 This line is printed by the `loopback` app on boot. It contains the device ID of the dongle, a 64-bit unique identifier (so everyone will see a different number); the radio channel that the device will use to communicate; and the transmission power of the radio in dBm.
 
+## USB issues
```

Comment:
not sure if this belongs here or in the troubleshooting chapter (plus adding a link to the troubleshooting chapter here in this case of course) ?

Reply:
yeah, I think putting this in the troubleshooting section would reduce clutter. Good call.